build.gradle

```diff
@@ -154,6 +158,22 @@ subprojects {
 
   task systemTestLibs(dependsOn: jar)
 
+  tasks.create(name: "copyDependantLibsJava", type: Copy) {
+    from(configurations.runtime) {
+      include('*.jar')
+      exclude('*scala*')
```

Comment:
I am not sure if this is reliable. It may be tough to separate the scala and java jars. I guess it may work if you included the patter "_$versions. baseScala" which resolves to _2.10 or _2.11.